bitnami/airflow-worker/2/debian-11/rootfs/opt/bitnami/scripts/libairflowworker.sh

```diff
@@ -46,6 +46,11 @@ airflow_worker_validate() {
 #   None
 #########################
 airflow_worker_initialize() {
+    # Remove airflow-worker.pid file if exists to prevent error after WSL restarts
+	if [ -f /opt/bitnami/airflow/tmp/airflow-worker.pid ]; then
```

Comment:
Could you fix the indentation of the if block?

Reply:
It seems an issue using tabs VS spaces, lets fix it in the file directly so you can continue with the review and any internal test